packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -138,15 +151,19 @@ function* loadTdoSaga(widgetId, tdoId) {
       // add bulk-edit-transcript engine if one was run
       .map(engineRun => {
         const engineId = get(engineRun, 'engine.id');
-        if ((engineId === 'bde0b023-333d-acb0-e01a-f95c74214607' || engineId === 'bulk-edit-transcript') && !engineRun.engine.category) {
+        if (
+          (engineId === 'bde0b023-333d-acb0-e01a-f95c74214607' ||
```

Comment:
not a fan of hard-coding ids but @gitrite mentioned this is temp but this needs to be changed in the near future